template/.github/ISSUE_TEMPLATE/publish_component.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Publish Component
+about: Create a ticket to drive compliance before publishing a a new componet to Auro Design System
+title: 'auro-[name]; publish to Auro Design System'
+labels: 'Status: Proposal, not-reviewed'
```

Comment:
Is the idea that either you or I would use this? If that is the case, then the default label should be something different I assume.

Reply:
I assume that the component developer would create this ticket and largely use it as a checklist themselves. Certain list items obviously requiring them to coordinate with either of us / a designer in order to complete.

Reply:
```suggestion
labels: 'Status: Review Needed, not-reviewed'
```